docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -4675,7 +4675,7 @@ Remove a node [`id`] from the swarm.
 ### Update a node
 
 
-`POST /nodes/(id)/update`
+`POST /nodes/(id or name)/update`
 
 Update the node `id`.
```

Comment:
Perhaps simplify to "Update a node" here as well, or "Update a node, identified by `id` or `name`", but possibly that's too verbose